tests/ElementalAreaTest.php

```diff
@@ -78,8 +78,8 @@ public function testGetOwnerPage()
         // OwnerClassName set
         $ownerpage2 = $area2->getOwnerPage();
 
-        $this->assertEquals("DNADesign\Elemental\Tests\Src\TestPage", $ownerpage1);
-        $this->assertEquals("DNADesign\Elemental\Tests\Src\TestPage", $ownerpage2);
+        $this->assertInstanceOf(TestPage::class, $ownerpage1);
+        $this->assertInstanceOf(TestPage::class, $ownerpage2);
```

Comment:
This was always just a bad test that relied on `ModelData::__toString()` returning the class name.